installer/I02_custom_os.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# EMBA - EMBEDDED LINUX ANALYZER
+#
+# Copyright 2020-2022 Siemens AG
+# Copyright 2020-2022 Siemens Energy AG
+#
+# EMBA comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+# welcome to redistribute it under the terms of the GNU General Public License.
+# See LICENSE file for usage of this software.
+#
+# EMBA is licensed under GPLv3
+#
+# Author(s): Benedikt Kuehne
+
+# Description: Helper/preinstaller module for (unix)-OS other than kali
+# 
+
+I02_custom_os() {
```

Comment:
should we rename this module to I02_ubuntu()?
Then we have the possibility to include further modules for other OSes